lib/puppet-languageserver/puppetfile/validation_provider.rb

```diff
@@ -106,6 +106,32 @@ def self.validate_resolution(puppetfile_document, document_uri, cache, module_pa
         end
       end
 
+      def self.find_dependencies(content)
```

Comment:
Is this method named ok?  As it's only getting the root forge modules, they're not dependencies as such

Reply:
This is ok for now.  It's a private implementation detail.